src/Main.java

```diff
@@ -0,0 +1,137 @@
+import java.util.Random;
+import java.util.Scanner;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        Main obj = new Main();
+
+        System.out.println("Метод findSymbolOccurance");
+        System.out.println("Символ зустрічається - " + obj.findSymbolOccurance("Java", 'a') + " раз/разів");
+
+        System.out.println();
+
+        System.out.println("Метод findWordPosition");
+        System.out.println("Result : " + obj.findWordPosition("Apollo", "pollo" ));
+        System.out.println("Result : " + obj.findWordPosition("Apple", "Plant" ));
+
+        System.out.println();
+
+        System.out.println("Метод stringReverse");
+        System.out.println("Result : " + obj.stringReverse("Hello"));
+
+        System.out.println();
+
+        System.out.println("Метод isPalindrome");
+        System.out.println(obj.isPalindrome("ERE"));
+        System.out.println(obj.isPalindrome("Allo"));
+
+        System.out.println();
+        System.out.println();
+
+        System.out.println("Гра - вгадай слово");
+        obj.guessWord();
+    }
+
+    public int findSymbolOccurance(String str, char ch) {
+
```

Comment:
Do not leave empty lines, please remove them and always apply formatting to your code, IDEA can help you with that just press CTRL+ALT+L on a class and that's it.